packages/server-core/knexfile.ts

```diff
@@ -0,0 +1,46 @@
+import fs from 'fs'
+import type { Knex } from 'knex'
+import path from 'path'
+
+import appConfig from '@etherealengine/server-core/src/appconfig'
+
+const migrationsDirectories = ['./migrations']
```

Comment:
could we replace this with a "find recursive" such that we can put each migration file along with it's service in a migrations folder, otherwise this folder will get big very quickly

Reply:
discussed this, not a good idea considering it will be far easier to see all the changes if they are in one place